src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -573,7 +580,18 @@ private static void addMusicBrainzIDs(File file, DLNAMediaAudio currentAudioTrac
 				currentAudioTrack.setMbidTrack(val.equals("") ? null : val);
 			}
 		} catch (Exception e) {
-			LOGGER.trace("Audio Tag not parsed: " + e.getMessage());
+			LOGGER.trace("audio musicBrainz tag not parsed: " + e.getMessage());
+		}
+	}
+
+	private static void addRating(AudioFile af, File file, DLNAMediaAudio currentAudioTrack) {
```

Comment:
Can this be `addAudioTrackRating`? Because we are in a big file that does much more than just audio files

Reply:
Sure ... good idea.